File: app/src/main/java/sugtao4423/calcsalary/SQLHelper.java
package sugtao4423.calcsalary;

import android.content.Context;
import android.database.sqlite.SQLiteDatabase;
import android.database.sqlite.SQLiteOpenHelper;

public class SQLHelper extends SQLiteOpenHelper{

    public SQLHelper(Context context){
        super(context, "database", null, 1);
    }

    @Override
    public void onCreate(SQLiteDatabase db){
        db.execSQL("CREATE TABLE database(hour, minute, changed, jikyu, memo)");
    }

    @Override
    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion){
    }

}

File: app/src/main/java/sugtao4423/calcsalary/SQLHelper.kt
package sugtao4423.calcsalary

import android.content.Context
import android.database.sqlite.SQLiteDatabase
import android.database.sqlite.SQLiteOpenHelper

class SQLHelper(context: Context) : SQLiteOpenHelper(context, "database", null, 2) {

    override fun onCreate(db: SQLiteDatabase?) {
        db!!.execSQL("CREATE TABLE salaries(wage INTEGER, minutes INTEGER, changed TEXT, memo TEXT)")
    }

    override fun onUpgrade(db: SQLiteDatabase?, oldVersion: Int, newVersion: Int) {
        if (oldVersion == 1) {
            onCreate(db!!)
            db.execSQL("INSERT INTO salaries SELECT jikyu, hour*60+minute, changed, memo FROM 'database'")
            db.execSQL("DROP TABLE 'database'")
        }
    }

}